### DISCUSSION TITLE
Feature Request - Separate 'action-deployer-openshift' Into Templating & Deployment Libraries

### DISCUSSION THREAD

**Discussion by cberg-aot** [bcgov/quickstart-openshift#989] [Ideas]
Just good practice to separate the logical function of these libraries and will make each more modular and extensible

**Comment by cberg-aot** [comment]
Forgot to add, try do decouple deployment from templating logic.  I.e expect a finished yaml to be the input..or possibly a helm chart

**Reply by DerekRoberts** [reply]
Any idea how to structure this?  Currently, they're mostly OpenShift oc commands surrounded by bash handling.

https://github.com/bcgov-nr/action-deployer-openshift/blob/main/action.yml

**Comment by cberg-aot** [comment]
Yeah,I Definitely want to have a chat!  Today or tomorrow, what's your
schedule like?

**Reply by DerekRoberts** [reply]
Let's get an idea hammered out!  Plan -> Teams chat!
